src/components/graph/ExerciseTreeGraph.tsx

```diff
@@ -295,6 +298,64 @@ export class ExerciseTreeGraph extends React.Component<ExerciseTreeGraphProps> {
   }
 }
 
+function drawNode(
+  nodeSelection: d3.Selection<
+    SVGGElement,
+    ExerciseGraphNode,
+    SVGGElement,
+    unknown
+  >,
+  layout: ExerciseGraphLayout,
+  shapeOptions: ShapeOptions
+): void {
```

Comment:
You could consider returning the node selection, making this chainable. You don't need it now tho 🤷‍♂️ .

Reply:
For the D3 _selection.call_ method, it never returns the result of this function, it always returns the original selection.  So I'm not sure it'll ever get used that way.

Reply:
Right, then you don't need it :)